### PR TITLE
Improve headings for new website

### PR DESCRIPTION
**Pull request by prince-chrismc**
## Description

The headers a used to autogenerated the title and table of contents, when they are missing the pages looks odd

Testing everything out over on https://prince-chrismc.github.io/pyrsia-website/ since that was easy for me

## Screenshots (optional)

image

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).
- [x] I've requested a review  from `pyrsia/collaborators`.

### CODE REVIEW

**Comment by prince-chrismc** [comment]
Do we hold off on https://github.com/pyrsia/pyrsia.github.io/pull/28 since there's a lot of review that's required

File: docs/tutorials/setup_on_macosx.md [hunk 1]
```diff
@@ -1,141 +1,170 @@
...
 Notice that node A and node B have now connected as peers and are able to communicate with each other. Verify that the PeerId you see here is the same as that for node A.
 
 ### Good news
+
 If you saw the above lines in your logs and did not see any failure/error messages your Pyrsia node network has now been setup.
-Also that means we did not break anything ;P.
+Also that means we did not break anything 😜
 
-You should now be able to interact with the Pyrsia node.
+You should now be able to interact with the Pyrsia Pode.
```
Comment (fishseabowl) [inline docs/tutorials/setup_on_macosx.md:69]:
```suggestion
You should now be able to interact with the Pyrsia Node.
```

File: docs/tutorials/setup_on_macosx.md [hunk 2]
```diff
@@ -1,141 +1,170 @@
...
-On a fresh terminal, navigate to PYRSIA_HOME. We will now configure the pyrsia_cli to connect with Node B.
-```
-cd target/debug
-./pyrsia config --add
+
+On a fresh terminal, navigate to `PYRSIA_HOME`. We will now configure the `pyrsia_cli` to connect with Node B.
+
+```sh
+$ cd target/debug
+$ ./pyrsia config --add
```
Comment (fishseabowl) [inline docs/tutorials/setup_on_macosx.md:77]:
````suggestion
cd target/debug
./pyrsia config --add
```
```
````

**Review by chb0github — APPROVED** [review]

**Review by fishseabowl — APPROVED** [review]
LGTM 👍

**Pull request merged**